virttest/utils_backup.py

```diff
@@ -391,3 +396,39 @@ def clean_checkpoints(vm_name, clean_metadata=True, ignore_status=True):
             else:
                 virsh.checkpoint_delete(vm_name, checkpoint,
                                         ignore_status=ignore_status)
+
+
+def enable_inc_backup_for_vm(vm, libvirt_ver=(99,99,99)):
+    """
+    For now, libvirt doesn't enable incremental backup by default. We
+    need to edit vm's xml to make sure it's supported.
+
+    :param vm: The vm to be operated
+    :param libvirt_ver: Since which libvirt version, we don't need to edit xml.
+    We need to track bz1799015 and update the default (99,99,99) when it's fixed.
```

Comment:
LGTM. 
I'd like to share a tip to you.  I usually add 'TODO' to the items that may need to be updated in the future. 
Filtering in the TO-DO tree would be a huge benefit. For the details, please refer to https://github.com/avocado-framework/avocado-vt/blob/master/virttest/libvirt_xml/capability_xml.py#L102

Reply:
thanks for the tips, updated.